pkgdb/src/resolver/lockfile.cc

```diff
@@ -425,6 +427,141 @@ LockfileRaw::clear()
 
 
 /* -------------------------------------------------------------------------- */
+void
+LockfileRaw::load_from_content( const nlohmann::json & jfrom )
+{
+  unsigned version = jfrom["lockfile-version"];
+  debugLog( nix::fmt( "lockfile version %d", version ) );
+
+  switch ( version )
+    {
+      case 0: *this = jfrom; break;
```

Comment:
quib: didnt expect that `LockfileRaw` is really just a json document 🤔

Reply:
Well, it's not.  It's the de-serialized version of it.  But the fancy `from_json` modeling and the `nlohmann::json` library gives us that functionality to automatically de-serialize if all the appropriate models are provided and in scope.   I'm trying a different syntax to help avoid confusion like this.  Hidden _cool_ stuff like that can be fun for the author, but usually no-one else.  Changing this to use the `get_to` syntax to avoid confusion.